.all-contributorsrc

```diff
@@ -1,994 +0,0 @@
-{
```

Comment:
just keep the file?

Reply:
yeah i guess let's keep it and just update the content later.

Also should I start making the changes u told in review?

Reply:
I don't have time to do a full review. but the tests are failing somehow, can you take a look?

Reply:
Sure I will see to it.

Reply:
can you please revert the file?

Reply:
sure